docker-app/qfieldcloud/core/models.py

```diff
@@ -628,6 +629,39 @@ class Meta:
         verbose_name = "organization"
         verbose_name_plural = "organizations"
 
+    def billable_users(self, from_date: date, to_date: Optional[date] = None):
+        """Returns the queryset of billable users in the given time interval.
+
+        Bilable users are users triggering a job or pushing a delta on a project owned by the organization.
+
+        Args:
+            from_date (datetime.date): inclusive beginning of the interval
+            to_date (Optional[datetime.date], optional): inclusive end of the interval (if None, will default to the last day of the month of the start date)
+        """
+
+        if to_date is None:
+            to_date = from_date.replace(
+                day=calendar.monthrange(from_date.year, from_date.month)[1]
+            )
+
+        users_with_delta = Delta.objects.filter(
+            project__in=self.projects.all(),
+            updated_at__gte=from_date,
+            updated_at__lte=to_date,
+        ).values_list("created_by_id", flat=True)
```

Comment:
Not sure it makes a difference (values_list still is a queryset, not an actual list, so it's not retrieved by Django). But it can't hurt either (and am too lazy looking at the actual query produced), so ok with the change